filament/backend/src/vulkan/VulkanDriver.cpp

```diff
@@ -907,7 +913,7 @@ bool VulkanDriver::isStereoSupported() {
         case backend::StereoscopicType::INSTANCED:
             return mContext.isClipDistanceSupported();
         case backend::StereoscopicType::MULTIVIEW:
-            // TODO: implement multiview feature in Vulkan.
+            return true;
```

Comment:
Also generally we should follow the convention from the file, like do something like:

`bool VulkanDriver::isMultiviewSupported() {`

Reply:
> Why would performance be impacted if we don't use the extension? 

I am pretty sure I read that it could affect shader compilation as a whole, but, of course, I cannot find the reference anymore... I am asking Vulkan buffs right now.

> I don't I want a compile-time switch for stuff like this.

Ok, it is a misunderstanding then. I thought that's what the Filament team wanted.
It is easier for us to just activate the extension all the time (when it is present that is).
Assuming we find it has a performance impact even if not used, is there some kind of runtime config you'd want us to implement to turn multiview off?

Reply:
Actually, we do want it to be based on availability and configuration.  The config, however, is not a build flag but [this](https://github.com/google/filament/blob/main/filament/include/filament/Engine.h#L318).  
See my attempt here to enable clip space only when necessary: https://github.com/google/filament/commit/450644ccd557bae704fb99e4ae31bb919b65c701

Reply:
Got it, thanks for the reference links!

BTW, it seems the extension is enabled in Angle unconditionally, so it should not have any negative impact when unused.